src/data/Releases.ts

```diff
@@ -1,4 +1,6 @@
-const RELEASES = {
+type ReleaseDate = Date | { START: Date; END: Date };
+
+const RELEASES: { [key: string]: ReleaseDate } = {
```

Comment:
type here should just be `RELEASES: release` everything else should be in the external type